Subject: Transitive dependency workaround
 Gradle doesn't support POM relocation. Adding a rule to maven.rules doesn't
 help because gradle-debian-helper only applies its ruleset if the version 
 isn't already generic (i.e. 'debian' or ends in '.x').
Forwarded: not-needed
Author: Christopher Hoskin <christopher.hoskin@gmail.com>
Last-Update: 2016-09-24
--- a/build.gradle
+++ b/build.gradle
@@ -328,3 +328,11 @@
 apply from: 'gradle/deploy.gradle'
 apply from: 'gradle/TestColorOutput.gradle'
 
+subprojects {
+    configurations.all {
+        resolutionStrategy {
+	    force 'org.apache.commons:commons-lang3:debian'
+            force 'org.apache.bcel:bcel:debian'
+        }
+    }
+}
